css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
you should set `position: relative` here instead of on all of the #ids

It technically does the same thing... but it's less typing you have to write since `.pokemon > img` is selecting all the img tags at the same time!